test/e2e_node/pods_container_manager_test.go

```diff
@@ -204,6 +221,15 @@ var _ = framework.KubeDescribe("Kubelet Cgroup Manager", func() {
 					err := framework.WaitForPodSuccessInNamespace(f.ClientSet, pod.Name, f.Namespace.Name)
 					Expect(err).NotTo(HaveOccurred())
 				})
+				By("Checking if the container cgroup was created", func() {
+					cId, err := parseContainerId(guaranteedPod)
```

Comment:
the CRI does not actually assert that the container id is required to correspond to the container cgroup name.

as a result, i do not think this test is portable, and the fact that it passes here is just a consequence of an implementation in one CRI.   please confirm @runcom @mrunalp @yujuhong that I am not mistaken.

i am not sure i see a major value in verifying the presence of the container cgroup versus other approaches in other tests like verifying container resource usage is charged back to the pod cgroup.  that approach verifies the core requirement better i think.

Reply:
done, Thanks